com.unity.render-pipelines.high-definition/Documentation~/Decal-Projector.md

```diff
@@ -45,6 +45,9 @@ Using the Inspector allows you to change all of the Decal Projector properties,
 
 ## Limitations
 
+- Decal Projector can affect Opaque Material with both a [Decal Shader](Decal-Shader.md) or a [Master Node Decal](Master-Node-Decal.md). It can affect Transparent Material only with a Decal shader. 
+- Decal Projector support [Decal Layers](Decal.md)
```

Comment:
- The Decal Projector supports [Decal Layers](Decal.md).

Although I'm not sure this is a limitation?

Could add:
The Decal Projector also supports Decal Layers which means you can control which Materials receive decals on a Layer by Layer basis.

to the end of the introduction instead of having it here?

Reply:
yes, good point. will do that